npa.js

```diff
@@ -283,7 +283,7 @@ function fromAlias (res, where) {
 
 function fromRegistry (res) {
   res.registry = true
-  const spec = res.rawSpec === '' ? 'latest' : res.rawSpec
+  const spec = res.rawSpec === '' ? 'latest' : res.rawSpec.trim()
```

Comment:
will this always be a string?

Reply:
Yes, the object that `npa` builds will always put a string here.  You can manually change it but that's invalid to begin with, there's no need to support it.